packages/gatsby-theme-store/src/sdk/masterData/README.md

```diff
@@ -0,0 +1,153 @@
+# Master Data
+The Master Data API documentation is based on the following schema:
+
+```ts
+{
+  name: { type: "string" },
+  email: { type: "string" },
+}
+```
+
+## mastData
+
+This section describes the `masterData` constructor and its parameters.
+
+### Properties
+
+| Property | Required |
+|----------|----------|
+| entity   | true     |
+
+Example
+
+```ts
+import masterData from '@vtex/gatsby-theme-store'
+
+const md = masterData('NL')
+```
+
+> `NL` is the entity
+
+
+## Queries
+
+### create
+
+`create` a new document.
+
+#### Example
+
+```ts
+const document = await md.create({
+  name: 'Emerson',
+  email: 'emerson@example.com',
+})
+```
+
+### upsert
+
+`upsert` query updates a document if it exists, or creates a new.
+
+#### Example
+
+Suppose we have this document:
+```ts
+{
+  id: "document-generated-id",
+  name: "Emerson",
+  email: "emerson@example.com"
+}
+```
+
+So let's do an `upsert`:
+```ts
+const document = await md.upsert({
+  id: "document-generated-id"
+  email: "laurentino@example.com"
+})
+```
+
+The document will now look like this:
+```ts
+{
+  id: "document-generated-id",
```

Comment:
I think people will just copy this code and everyone will be saved in the literal id of `document-generated-id`. Maybe try using uuid ?

Reply:
I believe that not, I decide to use `document-generated-id` precisely so that the user understands that there will be a UUID generated by the system (as in any database).

Reply:
ok 👍